lib/xboxkrnl/xboxkrnl.h

```diff
@@ -2111,6 +2080,8 @@ XBAPI VOID NTAPI RtlInitializeCriticalSection
     IN PRTL_CRITICAL_SECTION CriticalSection
 );
 
+#define RtlDeleteCriticalSection(CriticalSection) ((void)CriticalSection)
```

Comment:
A bit torn about including non-exports here; fine for now though.

Probably deserves a comment as to why this exists the way it does.

You might want to add a comment in the user though.
If you have confirmed behaviour in MS XAPI that should also be documented in the user.

Reply:
On the Xbox, `RtlInitializeCriticalSection` only sets up stuff in the `RTL_CRITICAL_SECTION` you pass, that's why no `RtlDeleteCriticalSection` exists as an export - there simply doesn't need to be one. Afaik XAPI headers define `RtlDeleteCriticalSection` and `DeleteCriticalSection` as `((void)0)`.
I decided to put `RtlDeleteCriticalSection` there for compatibility, in case some code out there expects this to exist.